.github/dependabot.yml

```diff
@@ -11,3 +11,9 @@ updates:
     groups:
       development-dependencies:
         dependency-type: "development"
+    ignore:
+      - dependency-name: "@react-native-async-storage/async-storage"
+      - dependency-name: "expo"
+      - dependency-name: "react-native"
+      - dependency-name: "react-native-safe-area-context"
+      - dependency-name: "react-native-screens"
```

Comment:
Curious - can we ignore only `major` (and maybe `minor`) version for some of these dependencies? (with `update-types` [option](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore))

Reply:
We could but wouldn't that break expo doctor passing anyway?